WalletWasabi.Fluent/Views/Wallets/Send/SendFeeView.axaml

```diff
@@ -174,13 +178,14 @@
             <Setter Property="Margin" Value="18,10,18,0"/>
             <Setter Property="MinHeight" Value="150"/>
             <Setter Property="YAxisLogarithmicScale" Value="False"/>
-            <Setter Property="AreaMargin" Value="30,0,10,30"/>
+            <Setter Property="AreaMargin" Value="30,0,10,32"/>
             <Setter Property="AreaStrokeThickness" Value="2"/>
             <Setter Property="AreaStrokeLineCap" Value="Round"/>
             <Setter Property="AreaMinViableWidth" Value="40"/>
             <Setter Property="AreaMinViableHeight" Value="20"/>
             <Setter Property="Focusable" Value="True"/>
             <Setter Property="YAxisTitle" Value="Fee Rate (sat/vByte)"/>
+            <Setter Property="XAxisTitle" Value="Estimated Confirmation Time"/>
```

Comment:
IMO this text should be removed because it clutters the dialog.

```suggestion
```

![feee](https://user-images.githubusercontent.com/52379387/152890005-37ef2a8c-ac6a-40f8-967b-94091768fe31.PNG)

Reply:
@yahiheb #7079 specifically asks for that text, it was not there before this PR. 

OTOH, the change from tooltip to always-visible for both ETA and Fee Rate certainly makes the axis title less relevant

Reply:
> OTOH, the change from tooltip to always-visible for both ETA and Fee Rate certainly makes the axis title less relevant

That is exactly what I meant, so IMO we can just remove it.